sklearn/model_selection/_search.py

```diff
@@ -560,44 +583,91 @@ def _fit(self, X, y, labels, parameter_iterable):
         # Out is a list of triplet: score, estimator, n_test_samples
         n_fits = len(out)
 
-        scores = list()
-        grid_scores = list()
-        for grid_start in range(0, n_fits, n_splits):
-            n_test_samples = 0
-            score = 0
-            all_scores = []
-            for this_score, this_n_test_samples, _, parameters in \
+        self.n_candidates_ = int(n_fits / n_splits)
```

Comment:
Please use `n_fits // n_splits` if this is truly the best way to calculate this number